test-requirements.txt

```diff
@@ -4,4 +4,4 @@ testrepository
 testtools
 tox
 wheel
-yapf==0.8.0
+yapf==0.9.0
```

Comment:
> Just an idea, but as we are always using the latest version of yapf, maybe we do not need to pin it.

Yeah, I actually moved to pinning a couple revs ago, b/c YAPF moves so fast, depending on the version of YAPF you are using, you will get different results.  Felt it was safer to pin, so we all format exactly the same way.